open-source-terms.md

```diff
@@ -0,0 +1,259 @@
+# npm Open-Source Terms
+
+These npm Open Source terms of use (these _Terms_) govern access to
+and use of <https://www.npmjs.com> (the _Website_) as well as the
+"npm Public Registry" at <https://registry.npmjs.com> (the _Public
+Registry_). Both the Website and the Public Registry are operated by
+npm, Inc. (_npm_). The Website and the Public Registry are referred to
+together as _npm Open Source_ throughout these Terms.
+
+These npm Open Source Terms were last updated on November 9, 2015.
```

Comment:
I've also thought about pointing to version control history for this kind of stuff. However, I agree with @kemitchell that the most recent update date ought to be in there. There's no reason not to point to git history for prior versions, though, like:

> These npm Open Source Terms were last updated on **November 9, 2015**. Prior versions are available at <https://github.com/npm/policies/releases>.

(This implies that you'd start making releases when you publish updated policies.)